Gemfile.lock

```diff
@@ -0,0 +1,54 @@
+PATH
```

Comment:
I removed the Gemfile.lock as per http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Let's not add back in